### PR TITLE
Add newrelic_telemetry_sdk Recipe

### PR DESCRIPTION
**Pull request by TimPansino**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by TimPansino** [comment]
@conda-forge-admin, please ping conda-forge/help-python

**Comment by TimPansino** [comment]
Ready for review.

I assume the added `-python` to the folder name to match our github repo makes the resulting feedstock repo `newrelic-telemetry-sdk-python-feedstock`, yes?

**Comment by TimPansino** [comment]
Also, confirming that I'm willing to be a maintainer.

File: recipes/newrelic-telemetry-sdk-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4,!=3.5,!=3.6
+    - setuptools-scm
+    - pip
+  run:
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4,!=3.5,!=3.6
```
Comment (ocefpaf) [inline recipes/newrelic-telemetry-sdk-python/meta.yaml:23]: It is now wrong but, b/c we dropped py27 a long time ago, this is easier to read and equivalent to the many "!" there.

```suggestion
    - python >=3.6
    - setuptools-scm
    - pip
  run:
    - python >=3.6
```

**Pull request merged**